index.html

```diff
@@ -20,34 +20,160 @@
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+    <div class="card ard3 img1">
+      <div class="card_inside">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+    <div class="card card4 img1">
+      <div class="card_inside">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+    <div class="card card5 img1">
+      <div class="card_inside">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+    <div class="card card6 img1">
+      <div class="card_inside">
+      <h2 class="paragraph_card">Multi-Post Stories <br>Gain+Glory</h2>
+      <div class="stack">
+        <ul class="listw">
+          <li ><div class="ruby">Ruby on rails</div></li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+      </div>
+      <button id="button" type="button"><p>See Project</p></button>
+      </div>
+    </div>
+  </section>
+  <section class="about">
+    <div class="about_container">
+      <h2 class="text_about">About me</h2>
+      <p class="text2_about"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button type="button" id="button_resume"><p class="p_resume">Get My Resume</p></button>
+    </div>
+    <div class="skills_container">
+      <div class="card_skills">
+          <img src="./img/Icon Langguage.png" alt="" id="icons">
+          <h2 class="skill_h2">Languages</h2>
+          <div class="stack_skills1">
+            <ul>
+              <li>JavasScript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>CSS</li>
+            </ul>
+          </div>
```

Comment:
- [ OPTIONAL ] I noticed that your Languages, Skills, and Frameworks cards don't look completely like the Figma Design. Please fix the button designs of the tech stacks and add a background color similar to what is on the template so your design can look more like the template you chose :wink: 
- The Figma design
![image](https://user-images.githubusercontent.com/102699958/212458515-fc42fa17-4865-4a19-a005-4a5ff622f172.png)
- Your design
![image](https://user-images.githubusercontent.com/102699958/212458559-1c4659cf-d9c5-45cc-b142-1e7cad2f61f1.png)